### ISSUE TITLE
Which Rasbian image?!?

### ISSUE THREAD

**Issue by JonMurphy** [home-assistant/home-assistant.github.io#1930]
The [Raspberry Pi All-In-One Installer](https://home-assistant.io/getting-started/installation-raspberry-pi-all-in-one/) page points to a Raspbian download page.  There are two choices for download:
**Raspbian Jessie With Pixel
Raspbian Jessie Lite**

Which one is the right one to download?  If either is fine can this be stated on the [Raspberry Pi All-In-One Installer](https://home-assistant.io/getting-started/installation-raspberry-pi-all-in-one/) webpage?

Thank you!

**Comment by Landrash** [comment]
In general you should use the Lite image for use with HA if it's a stand alone instance.
There's also a prepared image for HA called Hassbian available if you would prefer that.

**Comment by JonMurphy** [comment]
Is the [Raspberry Pi All-In-One Installer](https://home-assistant.io/getting-started/installation-raspberry-pi-all-in-one/) the same as the Hassbian image?  I picked the Raspberry Pi All-In-One Installer since it seemed to be easier and I had watched the BRUH Automation video.

I use the Lite version for my other RaspPis.  But I wasn't sure what to pick for HA.

I still feel the recommendation for the Lite image should be added to this page.  (I don't mind adding the words if it appropriate for me to do so)

Thank you!
